hotline/users/migrations/0002_auto_20150731_1546.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
I'm going to delete all these migrations, and rebuild them, since we don't need 3 migration files to add a couple of fields.